src/pybind/mgr/dashboard/controllers/cluster_configuration.py

```diff
@@ -39,3 +39,18 @@ def get(self, name):
                 return self._append_config_option_values([option])[0]
 
         raise cherrypy.HTTPError(404)
+
+    def create(self, name, value):
```

Comment:
You are missing permissions check in this controller:

Should be:

`@ApiController('/cluster_conf', Scope.CONFIG_OPT)`

Instead of:
`@ApiController('/cluster_conf')`

Reply:
Fixed!